### DISCUSSION TITLE
Display timers on BubbleCard

### DISCUSSION THREAD

**Discussion by mike02spb** [Clooos/Bubble-Card#1013] [Questions about config, custom styles and templates] [answered]
Hello! Thank you for your work!
There are problem with display countdown timers in bubble cards. When enable attribute "remaining" bubble cards shows duration
Can you fix it?

**Comment by MrBearPresident** [comment, marked as answer]
I converted to question about config. 
This is actually a problem rooted to Home Assistant itself. 
You can see this when you observe the state in the developer tools of home assistant. 

Please have a look at #937 for the work around.


---

<sup>If this solves your issue, please mark this as solved and close it.</sup>
